src/patient-lab-details/patient-lab-details.test.tsx

```diff
@@ -334,6 +334,111 @@ describe('Patient lab details', () => {
       ).toBeInTheDocument(),
     )
   })
+
+  it('should populate based on property if a pending order is selected', async () => {
+    Object.defineProperty(window, 'localStorage', {value: localStorageMock})
+    when(usePatient)
```

Comment:
wrap this into a function as `usePatientMock()`

Reply:
We have moved these functions to the `beforeEach`